### PR TITLE
The paragraph about using currently matched parameters is a bit diffi…

### PR DESCRIPTION
**Pull request by alexivaylov**
|    Q          |   A
|-------------- | ------
| Documentation | yes
| Bugfix        | no
| BC Break      | no
| New Feature   | no
| RFC           | no
| QA            | no

### Description
The paragraph about using currently matched route parameters is a bit difficult to read and I propose that we edit it in this way. * The tutorial suggests that Laminas\EventManager\EventManagerInterface::setIdentifiers() method accepts a string but the actual code suggests that it only accepts an array.

### CODE REVIEW

File: docs/book/in-depth-guide/data-binding.md [hunk 1]
```diff
@@ -317,10 +317,10 @@ The three differences between the `add` and `edit` templates are:
...
 passed. The way we've done this in the controllers is to pass the identifier as
 a parameter: `$this->url('blog/edit/', ['id' => $id])`. This would require that
-we pass the original `Post` instance or the identifier we pull from it to the
-view, however. laminas-router allows another option, however: you can tell it to
-re-use currently matched parameters.  This is done by setting the last parameter
-of the view-helper to `true`: `$this->url('blog/edit', [], true)`.
+we pass the original `Post` instance (or the identifier we pull from it) to
+the view. However, laminas-router allows another option: we can tell it to
+re-use the currently matched parameters. This is done by setting the last
+parameter of the view-helper to `true`: `$this->url('blog/edit', [], true)`.
```
Comment (froschdesign) [inline docs/book/in-depth-guide/data-binding.md:323]: These changes are unrelated to the current topic of this pull request, please remove all theses changes. If the correction is still relevant then please create a new pull request.

Comment (alexivaylov) [reply docs/book/in-depth-guide/data-binding.md]: @froschdesign I have reverted those changes as requested. I will be making a separate PR for them because I feel that paragraph is a bit difficult to read.

**Comment by michalbundyra** [comment]
Thanks, @alexivaylov!
